Update guide for inproc collector (#535)
tonerdo/coverlet: README.md

#### Requirements

> * _You need to be running .NET Core SDK v2.2.300 and above_

Author: ```suggestion
* _You need to be running .NET Core SDK v2.2.300 or newer_
```

> * _To run fully featured collectors([fixes this long standing bug](https://github.com/tonerdo/coverlet/issues/110)) you need to be running  .NET Core SDK v2.2.401 and above_

Reviewer: ```suggestion
* _To run fully-featured collectors ([see #110](https://github.com/tonerdo/coverlet/issues/110)) you need to be running  .NET Core SDK v2.2.401 and above_
```
